google-http-client/src/test/java/com/google/api/client/http/HttpResponseTest.java

```diff
@@ -58,6 +58,8 @@ public void testParseAsString_none() throws Exception {
 
   private static final String SAMPLE = "123\u05D9\u05e0\u05D9\u05D1";
   private static final String SAMPLE2 = "123abc";
+  private static final String VALID_CONTENT_TYPE = "text/plain";
```

Comment:
Can we add another valid complex content type? Maybe something like `application/vnd.com.google.datastore.entity+json; charset=utf-8; version=v1; q=0.9`

Reply:
Okay, added one with parameters.

Reply:
Awesome, thanks!